src/errors/encrypted_media_error.ts

```diff
@@ -54,7 +54,7 @@ export default class EncryptedMediaError extends Error {
       | { keyStatuses?: IEncryptedMediaErrorKeyStatusObject[] | undefined }
       | undefined,
   ) {
-    super();
+    super(errorMessage(code, reason));
```

Comment:
As the `Error` constructor's argument and the `message` attribute are semantically meant to be the exact same thing (as e.g. Firefox seems to rely on `message` and Chrome on the `Error` constructor's argument now for some reason, I've been wondering (I've no real answer) whether it wouldn't be more logical, when there's at least one transformation like here, to just rely on a common variable instead of repeating the same code at both places.

I wouldn't have cared at all if those two places could in theory have different meaning, but here I'm guessing we want to tie them in every cases no?

Reply:
I am wondering why there is an attribute `message` on EncryptedMediaError class. 
It is inherited from Error and `this.message` is set by the constructor of Error. I think we don't need to set manually this.message

Reply:
As discussed, I'm OK with this